src/client/src/ui/blotter/BlotterContainer.tsx

```diff
@@ -29,16 +28,15 @@ class BlotterContainer extends React.Component<BlotterContainerProps, {}> {
   }
 
   public render() {
-    const trades = _.values(this.props.blotterService.trades).reverse()
+    const trades = this.props.blotterService.trades
     const openFin = this.context.openFin
+    const gridRows = _.values(trades).reverse()
+    const popoutClick = this.props.onPopoutClick(openFin)
     return (
       <div className="shell_workspace_blotter">
-        <Blotter trades={trades}
-                 currencyPairs={this.props.currencyPairs}
-                 canPopout={Environment.isRunningInIE()}
-                 size={this.props.size}
-                 isConnected={this.props.isConnected}
-                 onPopoutClick={this.props.onPopoutClick(openFin)}/>
+        <AgGridBlotter rows={ gridRows }
+                       onPopoutClick={popoutClick}
+                       canPopout={true}/>
```

Comment:
I see that you've removed the check for IE11, is it no longer necessary to restrict popouts on IE11 because we are using ag-grid?

Reply:
this is a good spot. no, this is not intended and will cause a regression. will fix. thanks